pkg/goDB/DBWorkManager.go

```diff
@@ -42,15 +42,18 @@ type DBWorkManager struct {
 	iface              string
 	workloads          []DBWorkload
 	numProcessingUnits int
+
+	logger log.Logger
 }
 
 func NewDBWorkManager(dbpath string, iface string, numProcessingUnits int) (*DBWorkManager, error) {
 	// whenever a new workload is created the logging facility is set up
-	if err := InitDBLog(); err != nil {
+	l, err := log.NewConsoleLogger()
```

Comment:
Kept to console on purpose here as well?

Reply:
Turns out I didn't push the most recent changes... All fixed in recent commit.

Reply:
Yeah, totally makes sense - I just saw the change from the "old way" and was curious!